src/main/java/org/mskcc/limsrest/util/Utils.java

```diff
@@ -393,6 +391,23 @@ public static Boolean getRecordBooleanValue(DataRecord record, String key, User
         return new DataRecord[0];
     }
 
+    /**
+     * Returns Datatype descendents of the input DataRecord
+     *
+     * @param record
+     * @param dataType
+     * @param user
+     * @return
+     */
+    public static List<DataRecord> getDescendantsOfType(DataRecord record, String dataType, User user) {
```

Comment:
Why does getDescendantsOfType() need a wrapper function?  I forget the default Sapio behavior.

Reply:
I don't like always having to do the same exception handling